components/clm/src/main/subgridRestMod.F90

```diff
@@ -58,13 +60,245 @@ subroutine subgridRest( bounds, ncid, flag )
     !------------------------------------------------------------------------
 
     if (flag /= 'read') then
-       call subgridRest_write_only(bounds, ncid, flag)
+       if (flag == 'define') then
+          call subgridRest_define_only(bounds, ncid, flag)
+       else
+          call subgridRest_write_only(bounds, ncid, flag)
+       end if
     end if
 
     call subgridRest_write_and_read(bounds, ncid, flag)
 
   end subroutine subgridRest
```

Comment:
Can you also add calls to start and stop timers around the three calls to `subgridRest_*`? E.g.:
```
use perf_mod, only : t_startf, t_stopf
...
call t_startf('subgridRest_define')
call subgridRest_define_only(bounds, ncid, flag)
call t_stopf('subgridRest_define')
```
It'd help isolate any imbalance among read, define and write in future. TIA.

Reply:
@amametjanov - thanks for the suggestion. I committed changes to do so, if you want to review them